docs/user/commandline_tools.rst

```diff
@@ -192,6 +192,21 @@ Example export commands::
       -a "user_name,buf=buffer(geom\, 2)" \
       --format csv -q "[[ user_name like `John%' ] AND [ bbox(geom, 22.1371589, 44.386463, 40.228581, 52.379581, 'EPSG:4326') ]]"
 
+.. note::
+
+    Use the `export-bin` command to export data in binary format.
+
+Example binary export command::
```

Comment:
You'll also need to update the list of geomesa commands that gets outputted if you don't provide a command (i.e. if you just type "geomesa"). This needs to be updated here and in the geomesa tools readme.